plugins/liip/repaircafe/controllers/Events.php

```diff
@@ -24,4 +30,50 @@ public function __construct()
         parent::__construct();
         BackendMenu::setContext('Liip.RepairCafe', 'main-menu-cafe', 'side-menu-events');
     }
+
+    public function onDuplicate()
+    {
+        $original_event_id = Input::get('event_id');
+        $original_cafe_id = Input::get('cafe_id');
+
+        $original_event_model = Event::find($original_event_id);
+        $cafe_model = Cafe::find($original_cafe_id);
```

Comment:
isn't it possible to get the cafe_id directly from the Event model? Like `$original_event_model->cafe_id`?